data-es2016plus.js

```diff
@@ -1350,11 +1350,11 @@ exports.tests = [
         }
       },
       {
-        name: 'Atomics.wake',
-        spec: 'https://tc39.github.io/ecma262/#sec-atomics.wake',
-        mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics/wake',
+        name: 'Atomics.notify',
+        spec: 'https://tc39.github.io/ecma262/#sec-atomics.notify',
+        mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics/notify',
         exec: function () {/*
-         return typeof Atomics.wake === 'function';
+         return typeof Atomics.notify === 'function';
```

Comment:
I believe if you change the test you have to retest all targets or clear all the results.

Reply:
In this case, i think engines with wake but not notify should have a flag explaining the difference.